### ISSUE TITLE
Add translations

### ISSUE THREAD

**Issue by OSwoboda** [Wegi/pogo-cruncher#29]
I'm german and I need to lookup the german names everytime I'm using pogo-cruncher.

**Comment by Wegi** [comment]
Will add that, but other features are having priority right now. Anybody who is capable of delivering a pull-request with translations, can feel free to do so. :)

**Comment by OSwoboda** [comment]
What would be the best way to add them?
Add them as attribute to pogo-cruncher/frontend/src/cruncher/data/pokemon.cljs?
:name                        "Bulbasaur"
:name-de                   "Bisasam"

**Comment by Wegi** [comment]
Yeah, that would be a good variant, that doesn't break anything. :)

**Comment by OSwoboda** [comment]
I realised it would be some work to merge the translations into the file. But I've found translations that are ready to use here https://github.com/AHAAAAAAA/PokemonGo-Map/tree/master/static/locales

**Comment by Wegi** [comment]
Thanks, that makes it possible to merge them in with an automated script. I'll write it over the weekend I think. :)
